taimen/config.json

```diff
@@ -89,21 +87,390 @@
       ],
       "product-bytecode": [],
       "forced-modules": [
+        "chre",
         "com.android.ims.rcsmanager",
-        "RcsService",
-        "netutils-wrapper-1.0"
+        "ese-ls-provision",
+        "ese-replay",
+        "libhidltransport.vendor",
+        "libhwbinder.vendor",
+        "libjson",
+        "libp61-jcop-kit.vendor",
+        "libsensorndkbridge",
+        "libtinyxml",
+        "netutils-wrapper-1.0",
+        "RcsService"
       ],
       "new-modules": [],
-      "dep-dso": [],
+      "dep-dso": [
+        "vendor/lib64/libsdsprpc.so",
+        "vendor/lib/libsdsprpc.so"
+      ],
       "BoardConfigVendor": [],
       "device-vendor": [
         "PRODUCT_PROPERTY_OVERRIDES += \\",
         "    ro.hardware.fingerprint=fpc \\",
+        "    ro.hardware.vulkan=adreno \\",
+        "    ro.hardware.egl=adreno \\",
         "    ro.gfx.driver.0=com.google.pixel.wahoo.gfxdrv \\",
         "    ro.oem_unlock.pst=/dev/block/platform/soc/1da4000.ufshc/by-name/misc \\",
-        "    ro.oem_unlock.pst_offset=6144"
+        "    ro.oem_unlock.pst_offset=6144\\",
+        "    ro.storage_manager.enabled=true"
```

Comment:
Should we add this here? It seems like it's something for a generic feature and not hardware specific.

Reply:
Sure, removed. It's in the upstream `vendor.img` `build.prop` and I was ambivalent about adding it to our configs also.

Reply:
Yeah I was sticking with just having anything that's hardware-specific here, but good catch, we should set that in our device/ or vendor/ or something.